manifests/beanstalkd_exporter.pp

```diff
@@ -88,23 +88,23 @@
   Optional[Hash] $scrape_job_labels       = undef,
 ) inherits prometheus {
   #Please provide the download_url for versions < 0.9.0
-  $real_download_url = pick($download_url,"${download_url_base}/download/${version}/${package_name}-${version}.${os}-${arch}.${download_extension}")
+  $real_download_url = pick($download_url,"${download_url_base}/download/${version}/${package_name}-${version}.${os}-${arch}")
 
   $notify_service = $restart_on_change ? {
     true    => Service[$service_name],
     default => undef,
   }
 
-  $options = "-listen-address ${exporter_listen} -config ${config} -mapping-config ${mapping_config} ${extra_options}"
+  $options = "-web.listen-address ${exporter_listen} -beanstalkd.address ${beanstalkd_address} ${extra_options}"
 
   file { $config:
-    ensure  => file,
+    ensure  => absent,
     content => $beanstalkd_address,
     before  => Prometheus::Daemon['beanstalkd_exporter'],
   }
 
   file { $mapping_config:
```

Comment:
Empty mapping config will result in startup error (at least on 1.0.5) - as currently the module does not provide any function to set the content and it is not required it was deleted.
Users who need the mapping still can use `extra_options` to enable it - and have to maintain the mapping file themself as before.